joomla/joomla-cms
# Add consistency in Permissions tabs

### Summary of Changes

Small PR

All items that allow permissions across the core: Articles, Menu, Fields (Articles, Contacts and Users) and Categories (Articles, Banners, Contacts, News Feeds and Field Groups in Articles, Contacts and Users) and all component Options have the last tab named "Permissions".

There is only two different: Modules and Global config, in case of the modules the tab is even called "Module Permissions" ...

![image](https://cloud.githubusercontent.com/assets/9630530/20147697/16ec8aba-a6a1-11e6-9ea1-4b712b260f2a.png)

![image](https://cloud.githubusercontent.com/assets/9630530/20147728/328008c4-a6a1-11e6-8197-41a573624052.png)

So, this PR makes modules and global config behave like all the others views in this matter, ie, always the last tab and always with "Permissions" name.

### Testing Instructions

Very simple test. (30 sec)
- Apply patch
- Go to global config and check the last tab is "Permissions"
- Create/Edit a module and check the last tab is "Permissions"

### Documentation Changes Required

None.
